server/util/body-parser.js

```diff
@@ -311,8 +311,9 @@ export function findWpImageGallery(node) {
             const img = galleryItem.childNodes[0].childNodes.find((node) => {
               return node.nodeName === 'img';
             });
-            const width = parseInt(getAttrVal(img.attrs, 'data-original-width'), 10);
-            const height = parseInt(getAttrVal(img.attrs, 'data-original-height'), 10);
+            const dimensions = JSON.parse(`[${getAttrVal(img.attrs, 'data-orig-size')}]`);
+            const width = dimensions[0];
+            const height = dimensions[1];
```

Comment:
Why do we do this change?

I only ask as what was there before seemed to have worked, and I'm reluctant to change things in the parser as this code is reliant on WP data and thus brittle.